File: code/dd114.pas
program Bai114;
var xM, yM: real;
  begin
  readln(xM,yM);
  if (xM > 0) and (yM > 0) then writeln('M thuoc goc phan tu thu I') 
    else if (xM < 0) and (yM > 0) then writeln('M thuoc goc phan tu thu II') 
        else 
        if (xM < 0) and (yM < 0) then writeln('M thuoc goc phan tu thu III') 
            else  writeln('M thuoc goc phan tu thu IV');
   end.

File: code/dd114.py
xA, yA = [int(x) for x in  input().split()]
if (xA > 0) and (yA > 0):
  print('GOC PHAN TU THU I')
else:
  if (xA < 0) and (yA > 0):
    print('GOC PHAN TU THU II')
  else:
      if (xA < 0) and (yA < 0):
        print('GOC PHAN TU THU III')
      else:
        print('GOC PHAN TU THU IV')